modules/tensor_mechanics/include/kernels/MaterialVectorBodyForce.h

```diff
@@ -29,8 +29,8 @@ class MaterialVectorBodyForce : public Kernel
   /// coordinate axis this Kernel acts on
   unsigned int _component;
 
-  /// coupled force density vector property
-  const MaterialProperty<RealVectorValue> & _force_density;
+  /// coupled body force vector property
```

Comment:
I'm not a fan of this change, because as far as I can see the units here are a _force density_ and not a _force_, or am I wrong?

Reply:
This was mostly just to be consistent with the property name.

Reply:
To clarify a bit: The current code in MaterialVectorBodyForce.C has a parameter called `body_force`. I was trying to make everything consistent with that. It is commonly referred to as a body force, even though the understanding is that it is a density. My opinion is that we should stick with that convention. If we want a different name, we should change the name of the parameter for the fracture integral code in this PR as well.